### PR TITLE
Codestyle is changed according to eslint utility recommendations.

### PR DESCRIPTION
**Pull request by elshershnev**
Code was changed by eslint utulity with --fix parameter and now looks much better.

### CODE REVIEW

File: loggers/logger.js [hunk 1]
```diff
@@ -7,131 +7,128 @@ const transports = require('../transports');
 class Logger {
```
Comment (ubcent) [inline loggers/logger.js]: Why do you need so many empty lines?

File: loggers/logger.js [hunk 2]
```diff
@@ -7,131 +7,128 @@ const transports = require('../transports');
...
-            else {
-                throw new Error('trasports must be an Array');
-            }
-        }
-        else {
-            this.transports = [new transports.Console()];
-        }
+            );
+          }
+        }.bind(this));
```
Comment (ubcent) [inline loggers/logger.js]: I'd recommend you using arrow function instead of binding context here

File: loggers/logger.js [hunk 3]
```diff
@@ -7,131 +7,128 @@ const transports = require('../transports');
...
+      if (typeof options.formatter !== 'function') {
+        throw new Error('formatter must be a function');
+      }
+      this.formatter = options.formatter;
+    } else {
+      this.formatter = function(loggerName, JSTime, level, logItemId, logItem) {
+        const time = new Date(JSTime).toTimeString();
+        const logItemParts = [loggerName.toUpperCase(), time, logItemId, level];
+        if (typeof logItem !== 'object') {
+          logItemParts.push(String(logItem));
```
Comment (ubcent) [inline loggers/logger.js]: What the difference between these branches? In general it's the same

File: loggers/logger.js [hunk 4]
```diff
@@ -7,131 +7,128 @@ const transports = require('../transports');
...
+    if (!(level in levels)) {
+      throw new Error('Invalid log level');
+    }
 
-            logItem = this.formatter(this.name, now, level, logItemId, logItem);
+    if (levels[level] <= levels[this.boundaryLevel]) {
+      const now = new Date().getTime();
+      let logItemId = now.toString(36);
+      logItemId += logItemId.split('').filter(() => {
+        return Math.random() - 0.5;
```
Comment (ubcent) [inline loggers/logger.js]: Why do u need return here? As it seems to me, this style is more suitable:
```javascript
logItemId += logItemId.split('').filter(() => Math.random() - 0.5);
```

**Review by ubcent — CHANGES_REQUESTED** [review]

File: loggers/logger.js (hunk 1 above)
Comment (elshershnev) [reply loggers/logger.js]: Fixed.

File: loggers/logger.js (hunk 2 above)
Comment (elshershnev) [reply loggers/logger.js]: Fixed.

File: loggers/logger.js (hunk 3 above)
Comment (elshershnev) [reply loggers/logger.js]: Fixed.

File: loggers/logger.js (hunk 4 above)
Comment (elshershnev) [reply loggers/logger.js]: Filter does't work if no returned value presented and I found a bug here! Changed code  to tologItemId += logItemId.split('').filter(() => { return (Math.random() - 0.5) < 0 })...

File: loggers/logger.js [hunk 5]
```diff
@@ -82,7 +76,7 @@ class Logger {
       const now = new Date().getTime();
       let logItemId = now.toString(36);
       logItemId += logItemId.split('').filter(() => {
-        return Math.random() - 0.5;
+        return (Math.random() - 0.5) < 0;
```
Comment (ubcent) [inline loggers/logger.js]: In a previous review I meant something like following:
```javascript
logItemId += logItemId.split('').filter(() => (Math.random() - 0.5) < 0)
```

**Review by ubcent — APPROVED** [review]

File: loggers/logger.js (hunk 5 above)
Comment (elshershnev) [reply loggers/logger.js]: Got it

**Pull request merged**